### PR TITLE
Fix EZP-24439: Implement ContentType view

### PR DESCRIPTION
**Pull request by lolautruche**
> https://jira.ez.no/browse/EZP-24439

Pretty straight forward.
HTML markup follows legacy one. I guess that it will be changed at some point :smile:

### CODE REVIEW

**Comment by lolautruche** [comment]
Review ping @dpobel @andrerom @bdunogier @pspanja @yannickroger

**Comment by andrerom** [comment]
> HTML markup follows legacy one. I guess that it will be changed at some point :smile:

yesh, it is a pity you and @dpobel did not spend some time to get you up to speed on pure, so we didn

File: Resources/views/ContentType/view_content_type.html.twig [hunk 1]
```diff
@@ -23,13 +24,66 @@
...
+                    "%date%": content_type.modificationDate|localizeddate(locale=app.request.locale),
+                    "%modifier%": modifier.login
+                })
+            }}
+            </p>
+        </div>
+
+        <div class="block">
+            <div class="left">
+                <div class="block">
```
Comment (andrerom) [inline Resources/views/ContentType/view_content_type.html.twig]: Ok, word for mouth, lets try. correct me if I'm wrong here @dpobel:

```diff
+        <div class="pure-g">
+            <div class="pure-u-1 pure-u-sm-1-2">
+                <div>
```

I'm assuming all `class ="blocks"` can skipped as this is not float based like legacy css was.

File: Resources/views/ContentType/view_content_type.html.twig [hunk 2]
```diff
@@ -23,13 +24,66 @@
...
+                    {{ content_type.nameSchema }}
+                </div>
+
+                <div class="block">
+                    <h6>{{ "content_type.url_alias_schema"|trans }}:</h6>
+                    {{ content_type.urlAliasSchema }}
+                </div>
+            </div>
+
+            <div class="right">
```
Comment (andrerom) [inline Resources/views/ContentType/view_content_type.html.twig]: `<div class="pure-u-1 pure-u-sm-1-2">`

File: Resources/views/ContentType/view_content_type.html.twig [hunk 3]
```diff
@@ -23,13 +24,66 @@
...
+                    <h6>{{ "content_type.default_availability"|trans }}:</h6>
+                    {{ content_type.defaultAlwaysAvailable ? "content_type.always_available"|trans : "content_type.no_always_available"|trans }}
+                </div>
+
+                <div class="block">
+                    <h6>{{ "content_type.default_children_sorting"|trans }}:</h6>
+                    {{ ("content_type.sort_field." ~ content_type.defaultSortField)|trans(domain="ezrepoforms_content_type") }} / {{ ("content_type.sort_order." ~ content_type.defaultSortOrder)|trans(domain="ezrepoforms_content_type") }}
+                </div>
             </div>
+            <div class="float-break"></div>
```
Comment (andrerom) [inline Resources/views/ContentType/view_content_type.html.twig]: not needed anymore

File: Resources/views/ContentType/view_content_type.html.twig [hunk 4]
```diff
@@ -23,13 +24,66 @@
 
 {% block content %}
     <section class="ez-serverside-content">
-        <div class="ez-table-data is-flexible">
-            <div class="ez-table-data-container">
+        <div class="context-information">
+            <p class="left modified">
```
Comment (andrerom) [inline Resources/views/ContentType/view_content_type.html.twig]: can skip `left`, no such class here and left is anyway default alignment on ltr ;)

**Comment by dpobel** [comment]
+1

**Comment by yannickroger** [comment]
+1 (a bit late but still)

**Pull request merged**